cmd/catchpointdump/file.go

```diff
@@ -439,7 +403,7 @@ func printKeyValue(writer *bufio.Writer, key, value []byte) {
 	fmt.Fprintf(writer, "%s : %v\n", pretty, base64.StdEncoding.EncodeToString(value))
 }
 
-func printKeyValueStore(databaseName string, outFile *os.File) error {
+func printKeyValueStore(databaseName string, stagingTables bool, fileHeader ledger.CatchpointFileHeader, outFile *os.File) error {
```

Comment:
Did you just add `fileHeader` for consistency among the print funcs? It's unused, isn't it?

Reply:
I confirm `fileHeader` is unused.  I do not recall why the parameter was added during debugging.  Unless @cce recalls, I think it's preferable to remove it.